#132.java

```diff
@@ -0,0 +1,48 @@
+/**
+132. Palindrome Partitioning II
+
+Given a string s, partition s such that every substring of the partition is a palindrome.
+
+Return the minimum cuts needed for a palindrome partitioning of s.
+
+For example, given s = "aab",
+Return 1 since the palindrome partitioning ["aa","b"] could be produced using 1 cut.
+**/
+
+
+/**
+ create a int array minCut to cache the minimum cuts needed for a palindrome partition for string from position j to the
+ end, initialize it with maximum number of possible cuts: length of substring - 1
+ 
+ minCut[0] is the minimum cuts needed for a palindrome partition for the entire string, namely what we want to return
+ 
+ create a 2-d boolean array to cache whether string is palindrome from position j to position i 
```

Comment:
Still unclear. So what I really want to know is for substring from j to i, will it be stored at [i][j] or [j][i]. You can't always assume [i][j] convention. Be clear.